lib/auth/auth_with_roles.go

```diff
@@ -999,6 +1000,10 @@ func (a *AuthWithRoles) Close() error {
 	return a.authServer.Close()
 }
 
+func (a *AuthWithRoles) Wait(context.Context) error {
+	return nil
```

Comment:
Might want to return an error here so the caller knows this is not something they can use.

```
return trace.BadParameter("not implemented")
```